js/modals.js

```diff
@@ -0,0 +1,85 @@
+
+
+const closePopUp = document.getElementsByClassName('popup_close');
+
+const openPopUp_1 = document.getElementById('popup1_open');
```

Comment:
исправить на акутальный стандарт es6+
(для всех остальных тоже)
```suggestion
const openPopUp_1 = document.querySelector('#popup1_open');
```
привести нейминг в соответсвие с конвенцией нейминга